src/python/grpcio/grpc/experimental/aio/_server.py

```diff
@@ -41,6 +42,13 @@ def __init__(self, thread_pool: Optional[Executor],
                  maximum_concurrent_rpcs: Optional[int],
                  compression: Optional[grpc.Compression]):
         self._loop = asyncio.get_event_loop()
+        if interceptors:
+            invalid_interceptors = [interceptor for interceptor in interceptors
+                                    if not isinstance(interceptor,
+                                                      ServerInterceptor)]
+            if invalid_interceptors:
+                raise ValueError('Interceptor must be ServerInterceptor, the '
+                                 f'following are invalid: {invalid_interceptors}')
```

Comment:
I think that we committed to not use f-strings, even though the minimal target python version targeted for now is 3.6, which supports f strings.

@lidizheng f-strings yes or not?

Reply:
In Python, yes; in Cython, no.

If possible, I would like to use this Python 3 feature (simple is better than complex).

Cython doesn't understand f-string if it is compiled with Python < 3.6. We have learnt this in through test failures.